tests/test_variance_plane.py

```diff
@@ -333,7 +333,7 @@ def setUp(self):
 
         # Set the random seed for reproducibility.
         random_seed = galsim.BaseDeviate(1905).raw() + 1
-        np.random.seed(random_seed)
+        self.np_rng = np.random.Generator(np.random.MT19937(4))
```

Comment:
I did this as a side project but couldn't find a seed that would work exactly like before and so ended up slightly relaxing the test tolerance.